### DISCUSSION TITLE
Vote skip long introductions in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#19545] [Ideas] [answered]
I've not found a discussion on this but there may be one I missed.
It would be very useful to be able to skip intros on maps with long intros in a multiplayer map as we sometimes end up waiting 3 minutes to play a 2 minute map or something. Thanks.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/14260

**Reply by localcake** [reply]
Yeah sorry I'm not sure how to delete this now

**Comment by localcake** [comment]
I found a discussion on it, nevermind. 14260
